internal/pkg/pluginengine/change.go

```diff
@@ -96,6 +101,14 @@ func execute(smgr statemanager.Manager, changes []*Change) map[string]error {
 		var err error
 		var returnValue map[string]interface{}
 
+		log.Info("tool raw changes: ", c.Tool.Options)
```

Comment:
Let's keep "logs" (not errors, to be specific) as a "full sentence", which means:

- first letter as uppercase
- period at the end

Same for logs below.

Reply:
Please also check log lines in other files.